src/messages/responses/cache-list-fetch.ts

```diff
@@ -0,0 +1,33 @@
+import {SdkError} from '../../errors/errors';
+import {ResponseBase} from './response-base';
+import {applyMixins, ErrorBody} from '../../errors/error-utils';
+
+export abstract class Response extends ResponseBase {}
+
+export class Hit extends Response {
+  private readonly _values: Uint8Array[];
```

Comment:
I know this is wip PR but are you planning to add an array of string values?

Reply:
The type here (`Uint8Array[]`) seems right. 

style nit: I see we sometimes use `_` as the prefix and sometimes we don't do that.

Reply:
@virratanasangpunth Seems one can't have a property and a method with the same name, so I went with _private style.

@poppoerika PHP doesn't. dotnet-incubating does. Sure, I'll add it.

Reply:
We're using the .NET SDKs (incubating soon to be merged into main) as our SDK foundation, so I think we want to follow the .NET SDK rather than the PHP SDK.
Thank you!